fjord/base/google_utils.py

```diff
@@ -31,6 +31,9 @@ def ga_track_event(params, async=True):
        testing/development data hosing your production data.
 
     """
+    if sys.argv and sys.argv[0].endswith('py.test'):
```

Comment:
Acting differently while under test seems like a good way to make hard to reason about tests. Is there another way to deal with this?